feat: complete events.Event docs (#35360)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/events/event/haslisteners/index.md

---
title: Event.hasListeners()
slug: Mozilla/Add-ons/WebExtensions/API/events/Event/hasListeners
page-type: webextension-api-event
browser-compat: webextensions.api.events.Event.hasListeners
---

{{AddonSidebar}}

> Check if any listeners have been registered to the event.

Reviewer: ```suggestion
Checks if any listeners are registered to the event.
```

## Syntax

```js-nolint
events.Event.hasListeners()
```

### Parameters

None.

### Return value

> A `boolean` value: `true` if any listeners have been registered to the event, `false` otherwise.

Reviewer: ```suggestion
A boolean value: `true` if any listeners are registered to the event. Otherwise, `false`.
```
